client/src/utils/Server.ts

```diff
@@ -58,6 +58,7 @@ const Server = {
     getRanking: (name: string) => Fetch<BrutesGetRankingResponse>(`/api/brute/${name}/ranking`),
     exists: (name: string) => Fetch<BrutesExistsResponse>(`/api/brute/${name}/exists`),
     rankUp: (name: string) => Fetch<never>(`/api/brute/${name}/rank-up`),
+    ascend: (name: string, choice: WeaponName | SkillName | PetName) => Fetch<never>(`/api/brute/${name}/ascend/${choice}`),
```

Comment:
I would rather have the different perks sent as separate fields, to allow to easier checks in the backend, something like `ascend: (name: string, data: { weapon?: WeaponName, skill?: SkillName, pet?: PetName })` and send this data as the POST body instead of the URL params